webapp/src/main/webapp/WEB-INF/conf/officeonline/configuration.xml

```diff
@@ -591,11 +591,11 @@
             </field>
             <field name="patterns">
               <collection type="java.util.ArrayList" item-type="java.lang.String">
-                <value>
-                  <string>/intranet/documents.*</string>
+               <value>
+                  <string>.*/documents.*</string>

```

Comment:
In this case we'll work for all possible prefixes what is more than we want to do. As it's a regexp, we could restrict for only `/intranet` or `/dw` in all such places.

Reply:
This filter will be deleted in following PR https://github.com/exo-addons/office-online/pull/31